UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -177,12 +177,21 @@ private List<FileDetail> GetFileDetails()
             { new FileDetail() { FileName = "V01X01.zip", Hash = "Testdata" } };
             return lstFileDetails;
         }
+
+        private List<FileDetail> GetLargeMediaFileDetails()
+        {
+            return new List<FileDetail>()
+            {
+                new FileDetail() { FileName = "M01X02.zip", Hash = "Testdata" },
+                new FileDetail() { FileName = "M02X02.zip", Hash = "Testdata"}
+            };
+        }
+
         private ResponseBatchStatusModel GetBatchStatusResponse()
         {
-            string batchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
             return new ResponseBatchStatusModel()
             {
-                BatchId = batchId,
+                BatchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272",
                 Status = "Committed"
             };
```

Comment:
do we test if the batch is not committed

Reply:
Resolved. Added for failed status.